helpers/twitter.ts

```diff
@@ -38,7 +38,7 @@ async function postTweet(status: string) {
 async function deleteTweet(id: string) {
```

Comment:
I had no idea that this was implemented. I wonder if it actually works as expected.

Reply:
After looking at the code, I have a hard time to believe it does. It seems to keep track of the posted tweets through a `json` file which I don't believe is reliable or even persisted.

Reply:
is this holding back the PR?